### PR TITLE
Drop EOL django versions

### PR DESCRIPTION
**Pull request by Surgo**
- [Django 1.8 and 1.10 supports are ended](https://www.djangoproject.com/download/#supported-versions)
- To support 2.0, drop support for EOL versions. #98

### CODE REVIEW

File: cachalot/utils.py [hunk 1]
```diff
@@ -31,29 +31,26 @@ class IsRawQuery(Exception):
...
 else:
     CACHABLE_PARAM_TYPES.update((
         Binary,
-        NumericRange, DateRange, DateTimeRange, DateTimeTZRange, Inet, Json))
-    if django_version[:2] >= (1, 11):
-        from django.contrib.postgres.fields.jsonb import JsonAdapter
-        CACHABLE_PARAM_TYPES.add(JsonAdapter)
+        NumericRange, DateRange, DateTimeRange, DateTimeTZRange, Inet, Json,
+        JsonAdapter, ))
+    UNCACHABLE_FUNCS.update((
```
Comment (Surgo) [inline cachalot/utils.py]: I think that required when use postgresql.

**Comment by BertrandBordage** [comment]
Thanks @Surgo! I’ll review it and try to issue a new 2.0 version by the end of next week :) I need it to migrate one of my projects, so you can be sure it will not be delayed any longer.

**Comment by BertrandBordage** [comment]
Hey, sorry, this will have to wait for an extra week, I planned on working on it this week-end, but something else came up.
I’m not forgetting it, I still strongly need it ;)

File: cachalot/utils.py [hunk 2]
```diff
@@ -31,29 +31,26 @@ class IsRawQuery(Exception):
...
-
-if PY2:
-    CACHABLE_PARAM_TYPES.add(long)
-
-UNCACHABLE_FUNCS = set()
-if django_version[:2] >= (1, 9):
-    from django.db.models.functions import Now
-    from django.contrib.postgres.functions import TransactionNow
-    UNCACHABLE_FUNCS.update((Now, TransactionNow))
+CACHABLE_PARAM_TYPES.update(integer_types)  # Add long for Python 2
```
Comment (BertrandBordage) [inline cachalot/utils.py]: No need to add it afterwards, it can be put directly in the set literal above.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -191,7 +191,7 @@ Examples:
 Template utils
 ..............
 
-`Caching template fragments <https://docs.djangoproject.com/en/1.8/topics/cache/#template-fragment-caching>`_
+`Caching template fragments <https://docs.djangoproject.com/en/1.11/topics/cache/#template-fragment-caching>`_
```
Comment (BertrandBordage) [inline docs/quickstart.rst]: Can even be 2.0 :)

Comment (BertrandBordage) [reply docs/quickstart.rst]: Nevermind sorry, I thought this pull request also introduced 2.0 support.

**Comment by BertrandBordage** [comment]
Excellent work @Surgo :)
Merging it right now.

File: cachalot/utils.py (hunk 2 above)
Comment (BertrandBordage) [reply cachalot/utils.py]: Nevermind, didn’t pay attention it was an iterable.

**Pull request merged**